apps/drupal-default/particle_theme/templates/block/block--gtranslate.html.twig

```diff
@@ -0,0 +1,58 @@
+{#
+/**
+ * @file
+ * Theme override to display a block.
+ *
+ * Available variables:
+ * - plugin_id: The ID of the block implementation.
+ * - label: The configured label of the block if visible.
+ * - configuration: A list of the block's configuration values.
+ *   - label: The configured label for the block.
+ *   - label_display: The display settings for the label.
+ *   - provider: The module or other provider that provided this block plugin.
+ *   - Block plugin specific settings will also be stored here.
+ * - content: The content of this block.
+ * - attributes: array of HTML attributes populated by modules, intended to
+ *   be added to the main container tag of this template.
+ *   - id: A valid HTML ID and guaranteed unique.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ *
+ * @see template_preprocess_block()
+ */
+#}
+{%
+  set classes = [
+    'container',
+    'pt-6',
+    'px-6',
+    'pb-0',
+    'flex',
+    'flex-row-reverse',
+    'block',
+    'block-' ~ configuration.provider|clean_class,
+    'block-' ~ plugin_id|clean_class,
+  ]
+%}
+
+<style>.gtranslate > select { 
```

Comment:
If I need how to learn how to do this properly, let me know.

Reply:
These rules give it a similar look to how it was done in OUC, minus the Google branding.

Reply:
Looks good to me. There's different ways of approaching it and it all depends on how much data, variables the twig files deal with. But for this specific scenario that looks fine to me.
So for something as large as a card I would not recommend it but this is very atomic.